src/controllers/hid/hiddenylist.h

```diff
@@ -1,21 +1,35 @@
 #pragma once
 
-typedef struct hid_denylist {
+// TODO: unify this with the invalid interfacenumber from the bulkenumerator
+constexpr static int kInvalidInterfaceNumber = -1;
+constexpr static unsigned short kBlanketValue = 0x0;
```

Comment:
> However there are devices which report the -1

Are you sure? that violates the assumption hidapi makes:
https://libusb.info/hidapi/structhid__device__info.html#a9163d8d5d7db8dc47bddfaf876e17547

Reply:
This reads as if non HID devices are -1. 
Somewhere else I have read that the interface number is Linux only. I don't understand the issue fully.

Reply:
No, this reads that non-USB devices are -1, because only USB interfaces can have a valid interface number. HID is a high level protocol, that can be used with many other transport protocols, which do not know the concept of enumerated interfaces.
To my knowledge, an USB device can not have any other interface number than 0...255. This is why uint8 is enough for the lookup table of USB bulk interfaces. Invalid is impossible in an enumeration and it's not a filter, where a don't-care would make sense.

Reply:
The limit seems to be 0...32.